### ISSUE TITLE
Permission denied error in Galaxy S7 (G930F)

### ISSUE THREAD

**Issue by ThallesAraujo** [vusec/drammer#15]
Device information:
Android Version: 8.0.0 Oreo
UX Version: Samsung Experience 9.0
Android Security Patch Level: April 1, 2018
(Occurs both in default and aggressive modes)
![screenshot_20180628-164006_drammer](https://user-images.githubusercontent.com/17062668/42057059-77d93f7c-7af2-11e8-80f7-61fefa4fbdba.jpg)

**Comment by vvdveen** [comment]
This may have been an issue in how we invoke the native binary from the app. I pushed an updated version of the apk on Friday morning (June 29) that should address this. Maybe you can remove the app and redownload it?

Thanks,
Victor

**Comment by ThallesAraujo** [comment]
Ok, I did it and I guess this time it worked. I was able to see the output.
I did the test with apps closed and rebooting the device, too (as recomended in output. This was a result after restarting).

![screenshot_20180701-175952_drammer](https://user-images.githubusercontent.com/17062668/42138760-e72c2a3e-7d58-11e8-9e0b-2830d9cefc9b.jpg)

**Comment by vvdveen** [comment]
Yes, this looks better. I have seen similar output before and it seems that (some) galaxy devices disabled access to some of the files in /proc/. This makes it harder for our app to figure out when to start hammering. I will have to think about how to address this. 

In general, I think that it is a good strategy from Samsung to disable read access to those files.
